assets/js/modules/ads/components/settings/SettingsView.js

```diff
@@ -91,18 +91,14 @@ export default function SettingsView() {
 										'Enhanced Conversion Tracking',
 										'google-site-kit'
 									),
-									status: isConversionTrackingEnabled
-										? __( 'Enabled', 'google-site-kit' )
-										: __( 'Disabled', 'google-site-kit' ),
+									status: isConversionTrackingEnabled,
 								},
 								{
 									label: __(
 										'First-Party Mode',
```

Comment:
@hussain-t @benbowler there's been a small change to the label text - as per this [thread in Figma](https://www.figma.com/design/udlPW35VZ0EDnSbRxYOGMj?node-id=30-1215#1045957772), the label will now be "First-party Mode".

It's probably worth updating the label for this and the Analytics view in this issue.

Reply:
I'll take it back into execution briefly while I update VRTs etc.